docs/user_guide/1_setting_up_the_hand.md

```diff
@@ -25,19 +25,21 @@ The most important one is to have a fast HDD or an SSD.
 ### What's in the box?
 
 ```eval_rst
-=========================   ===========================================================
-Item                        Description
-=========================   ===========================================================
-Shadow Hand E2M3 or E2PT    Hand Unit
-PC                          Host PC control unit for the hand
-PSU for Hand                48v for motor hand
-Kettle Leads                To connect power supplies to mains
-Power Cable                 4-pin Large Lemo connector, already fitted to the hand
-EtherCAT Extension Cable    50cm EtherCAT extension lead, already fitted to the Hand
-Calibration Jigs            Bag containing calibration jigs for all joints
-Toolbox                     Contains hex drivers to perform required maintenance
-User Manual                 This document
-=========================   ===========================================================
+=============================   ==========================================================
+Item                            Description
+=============================   ==========================================================
+Shadow Hand E2M3 or E2PT        Hand Unit
+NUC control machine             NUC minicomputer for running hand's driver
+USB->Ethernet adapter x3        Adapters for connections between NUC, hand and client PC
+PSU for Hand                    48v for motor hand
+Kettle Leads                    To connect power supplies to mains
+Power Cable                     4-pin Large Lemo connector, already fitted to the hand
+EtherCAT Extension Cable x2     50cm EtherCAT extension lead, already fitted to the Hand
+Calibration Jigs                Bag containing calibration jigs for all joints
+Toolbox                         Contains hex drivers to perform required maintenance
+User Manual                     This document
+Client PC (optional)            Host PC control unit for the hand
+=============================   ==========================================================
```

Comment:
Add: 64GB USB stick with Clonezilla backup copies of the NUC and optionally the client PC